src/server/rdb_load.cc

```diff
@@ -2369,27 +2392,29 @@ void RdbLoader::LoadScriptFromAux(string&& body) {
 void RdbLoader::LoadSearchIndexDefFromAux(string&& def) {
   facade::CapturingReplyBuilder crb{};
   ConnectionContext cntx{nullptr, nullptr, &crb};
+  cntx.is_replicating = true;
   cntx.journal_emulated = true;
   cntx.skip_acl_validation = true;
```

Comment:
not sure how journal emulated is not enough for replication, will look into it 😵 (we set all 3 in journal executor)